cli/internal/run/dry_run.go

```diff
@@ -304,6 +305,12 @@ func displayDryTextRun(ui cli.Ui, summary *dryRunSummary, workspaceInfos workspa
 		fmt.Fprintln(w, util.Sprintf("  ${GREY}Log File\t=\t%s\t${RESET}", task.LogFile))
 		fmt.Fprintln(w, util.Sprintf("  ${GREY}Dependencies\t=\t%s\t${RESET}", strings.Join(dependencies, ", ")))
 		fmt.Fprintln(w, util.Sprintf("  ${GREY}Dependendents\t=\t%s\t${RESET}", strings.Join(dependents, ", ")))
+
+		count := 0
+		for range task.ExpandedInputs {
+			count = count + 1
+		}
```

Comment:
Could this be `len(task.ExpandedInputs)`?

Reply:
🤦🏾 .  I don't know what happened here. I searched for how to do this and might have just ignored the results entirely...?

![CleanShot 2023-02-27 at 22 04 46@2x](https://user-images.githubusercontent.com/490968/221768150-e12f6276-1b3f-452f-8201-71775843b45f.png)